documentation/man/features/report.rst

```diff
@@ -39,6 +53,29 @@ OPTIONS
 \--year[=<year>]:
   Exhibits the current year summary if the user does not specify *<year>*.
 
+\--statistics:
+  Display current date statistics summary.
+
+\--statistics [--day[=<year>/<month>/<day>]]:
+  Display day statistics summary. Users have the option to search a specific
+  date by passing an argument that follows the *<year>/<month>/<day>* format.
+  Not passing anything and gets info about today.
+
+\--statistics [--week[=<year>/<month>/<day>]]:
+  Shows the week summary. If a user does not pass any parameter kw will show
+  the current week statistics. However, users can pass a random date
+  *<year>/<month>/<day>* and let **kw** take care to provide a summary
+  related to the week related to the target date.
+
+\--statistics [--month[=<year>/<month>]]:
+  Shis option shows a report regarding a specific month. Users can search for
+  data related to a specific month by providing a parameter in the
+  *<year>/<month>* format. If the user does not pass any parameter, **kw**
+  displays the current month data.
```

Comment:
The option `kw report --help` will looks like this for this section:
```
--day[=<year>/<month>/<day>]:
Display day summary. Users have the option to search a specific date by passing an argument that follows the <year>/<month>/<day> format. Not passing anything and gets info about today.

--week[=<year>/<month>/<day>]:
Shows the week summary. If a user does not pass any parameter kw will show the current week statistics. However, users can pass a random date <year>/<month>/<day> and let kw take care to provide a summary related to the week related to the target date.

--month[=<year>/<month>]:
This option shows a report regarding a specific month. Users can search for data related to a specific month by providing a parameter in the <year>/<month> format. If the user does not pass any parameter, kw displays the current month data.

--year[=<year>]:
Exhibits the current year summary if the user does not specify <year>.

--statistics:
Display current date statistics summary.

--statistics [–day[=<year>/<month>/<day>]]:
Display day statistics summary. Users have the option to search a specific date by passing an argument that follows the <year>/<month>/<day> format. Not passing anything and gets info about today.

--statistics [–week[=<year>/<month>/<day>]]:
Shows the week summary. If a user does not pass any parameter kw will show the current week statistics. However, users can pass a random date <year>/<month>/<day> and let kw take care to provide a summary related to the week related to the target date.

--statistics [–month[=<year>/<month>]]:
Shis option shows a report regarding a specific month. Users can search for data related to a specific month by providing a parameter in the <year>/<month> format. If the user does not pass any parameter, kw displays the current month data.

--statistics [–year[=<year>]]:
Exhibits the current year summary if the user does not specify <year>.
```
I don't think it makes sense to explain the date parameters repeatedly. Please, rewrite it by removing the duplication of information.